src/main/java/com/acrolinx/sidebar/jfx/AcrolinxSidebarPlugin.java

```diff
@@ -376,7 +377,7 @@ public synchronized void checkDocumentInBatch(final String documentIdentifier, f
                 jsObject.eval("acrolinxSidebar.checkDocumentInBatch(documentIdentifier, documentContent, "
                         + options.toString() + ");");
             } catch (final Exception e) {
-                logger.error(e.getMessage(), e);
+                logger.error("", e);
```

Comment:
I know it's redundant to provide the error message but maybe we can put some other message instead of an empty string?

Reply:
if we have any good idea what context variables or so should be logged too (to make debugging easier), I am all for it.

Reply:
We might leave it like that for now, since I have no better message atm